### PR TITLE
use ember-auto-import

### PR DESCRIPTION
**Pull request by hugurp**
Start using https://github.com/ef4/ember-auto-import

instead of the traditional `import {someThing} from npm:some-thing`

### CODE REVIEW

**Comment by hugurp** [comment]
I'm not sure what's going on but locally i've got errors.

```
ypeError: Array.prototype.slice.apply(...).filter(...)[0] is undefined[Learn More]
app.js%20line%2065834%20%3E%20eval:4:1
<anonymous>
webpack://__ember_auto_import__/./tmp/ember_auto_import_webpack-staging_dir-WEF1Z69M.tmp/app.js?:4:1
./tmp/ember_auto_import_webpack-staging_dir-WEF1Z69M.tmp/app.js
app.js:165
__webpack_require__
app.js:80
checkDeferredModules
app.js:47
webpackJsonpCallback
app.js:34
<anonymous>
app.js:170
```

**Comment by hugurp** [comment]
Ty! Will try this.
How did you figure it out? Could not find out

File: app/index.html [hunk 1]
```diff
@@ -25,9 +24,8 @@
 	<div id="Radio4000" class="Radio4000"></div>
 	<noscript>Sorry, to use Radio4000 you need to enable JavaScript in your browser.</noscript>
 
-	<!-- <script src="{{rootURL}}assets/vendor.js"></script> -->
-	<!-- <script src="{{rootURL}}assets/radio4000.js"></script> -->
-	{{content-for "concat-js"}}
+	<script src="{{rootURL}}assets/vendor.js"></script>
+	<script src="{{rootURL}}assets/radio4000.js"></script>
```
Comment (oskarrough) [inline app/index.html]: mind keeping this one as `async`?

Comment (hugurp) [reply app/index.html]: why keeping?
just the one on line 28?

Comment (oskarrough) [reply app/index.html]: Yes, just the second. So we're sure it loads after the first.

It should be faster when async, so it doesn't block rendering.

**Pull request merged**